### PR TITLE
test: use sourcemaps in unit tests

### PR DESCRIPTION
**Pull request by psmarshall**
Introduce the source-map-support package and require it for mocha
running unit tests.

Turn on the sourceMap option for tsconfig.base.json so that the
sourceMappingURL= line is emitted in the generated files.

Before:
Screen Shot 2020-10-08 at 1 10 45 pm

After:
Screen Shot 2020-10-08 at 11 58 04 am

### CODE REVIEW

**Review by jackfranklin — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -53,6 +53,7 @@
     "progress": "^2.0.1",
     "proxy-from-env": "^1.0.0",
     "rimraf": "^3.0.2",
+    "source-map-support": "^0.5.19",
```
Comment (mathiasbynens) [inline package.json]: Since Puppeteer users don’t need this dependency, can we move this to `devDependencies` instead?

File: tsconfig.base.json [hunk 2]
```diff
@@ -8,6 +8,7 @@
     "module": "ESNext",
     "declaration": true,
     "declarationMap": true,
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "sourceMap": true,
```
Comment (mathiasbynens) [inline tsconfig.base.json:12]: It triggers me that this is `.json` but actually supports trailing commas 🤯

**Review by mathiasbynens — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (psmarshall) [reply package.json]: I thought I did this but npm disagrees. Who's to say who is right and who is wrong...

File: tsconfig.base.json (hunk 2 above)
Comment (psmarshall) [reply tsconfig.base.json:12]: 😨

**Review by mathiasbynens — APPROVED** [review]

**Pull request merged**